.gitignore

```diff
@@ -23,3 +23,5 @@ packages/*/target/
 target/
 *.log
 packages/*/lib
+coverage
```

Comment:
Hmm, is there any part of the build process that writes to `./coverage`? Or are those reports caused by an IDE you use? (I don't mind having that ignored, just curious 😄)

Reply:
Removed - I think it came from VSCode on windows.